store/etcd.go

```
@@ -54,8 +54,7 @@ func (*etcdStore) Values(path string) (values map[string]string) {
 	values = map[string]string{}
 	resp, err := api.Get(context.Background(), path, &etcdclient.GetOptions{Recursive: true})
 	if err != nil {
-		log.Println("Ignoring error getting values:" + path)
-		log.Println(err)
+		glog.Errorf("Ignoring error getting values: %s. Error: %s", path, err.Error())
```

This is just an example of a line i was semi confused on... I would have chosen `Warn` here since it is ignoring the error